### ISSUE TITLE
Security: Don't package current dir

### ISSUE THREAD

**Issue by glensc** [ivanilves/lstags#234]
```
glen@lima /tmp$ tar xf lstags-linux-v1.2.21.tar.gz
tar: .: Cannot utime: Operation not permitted
tar: .: Cannot change mode to rwxr-xr-t: Operation not permitted
tar: Exiting with failure status due to previous errors

glen@lima /tmp$ tar tvf lstags-linux-v1.2.21.tar.gz
drwxr-xr-x circleci/circleci 0 2021-07-21 08:38 ./
-rwxr-xr-x circleci/circleci 9326592 2021-07-21 08:38 ./lstags

glen@lima /tmp$
```

Huh, glad that I wasn't `root` when unpacking!

**Comment by ivanilves** [comment]
Good one, will look into release script.

**Comment by ivanilves** [comment]
Fixed since `v1.2.23` :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/9897741/162564810-2f8306c3-34ce-4e03-922f-1128a676c0f0.png)
